### PR TITLE
Libreoffice upgrade

### PR DESCRIPTION
**Pull request by txapelgorri**

### CODE REVIEW

**Comment by txapelgorri** [comment]
Iñigo, maybe merge is not the best option. I was thinking on splitting the code in two main branches: those who need to use DocumentConvert.py and those who can upgrade LibreOffice in the server to a more recent version, but perhaps it's to silly to maintain the first "ancient" version of the code because many people can upgrade an ancient Hobo app (Ruby 1.8.7) to a more updated Ubuntu / Linux version (this is what I did with Inpromat's one), what do you think?.

File: lib/hobo-serenity.rb [hunk 1]
```diff
@@ -1,106 +1,91 @@
...
+    odt_path = document_path + '.odt'
+    pdf_path = document_path + '.pdf'
+
+    # Generate ODT with Serenity
+    render_odt template, odt_path
+    File.chmod(0664, odt_path)
+      
+    # Convert to PDF via LibreOffice 
+    if create_pdf == true
+      odt_to_pdf_cmd = "soffice --headless --convert-to pdf --outdir '#{[Rails.root, document_folder].join('/')}' '#{[Rails.root, odt_path].join('/')}' &>>#{[Rails.root,'log',Rails.env + ".log"].join('/')}"
```
Comment (iox) [inline lib/hobo-serenity.rb]: I had not seen this syntax before - it's great what LibreOffice does now :)

File: README.textile [hunk 2]
```diff
@@ -0,0 +1,66 @@
+h2. HoboSerenity
+
+This plugin tries to simplify the process of integrating two great gems: Hobo and Serenity-ODT. It also uses a LibreOffice to covvert automatically ODT template into PDF file if you want to.
```
Comment (iox) [inline README.textile]: There's a typo here: "to convert" :)

**Comment by iox** [comment]
Let's merge this: less code, nicer readme. Everything looks great.

**Pull request merged**